docs/DeveloperGuide.md

```diff
@@ -293,9 +293,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
```

Comment:
Maybe we can include dogs and programs in here too?

Reply:
okay i will add that!